pkgs/tools/graphics/grace/default.nix

```diff
@@ -0,0 +1,69 @@
+{ lib
+, stdenv
+, fetchurl
+, coreutils
+, motif
+, fftw
+, netcdf
+, libpng
+, libjpeg_original
+, libX11
+, libSM
+, libXext
+, libXmu
+, libXp
+, libXpm
+, libXt
+}:
+
+stdenv.mkDerivation rec {
+  pname = "grace";
+  version = "5.1.25";
+  src = fetchurl {
+    url = "ftp://ftp.fu-berlin.de/unix/graphics/grace/src/grace5/grace-${version}.tar.gz";
+    sha256 = "751ab9917ed0f6232073c193aba74046037e185d73b77bab0f5af3e3ff1da2ac";
+  };
+
+  buildInputs = [
+    coreutils
+    motif
+    fftw
+    netcdf
+    libpng
+    libjpeg_original
+    libX11
+    libSM
+    libXext
+    libXmu
+    libXp
+    libXpm
+    libXt
+  ];
+
+  enableParallelBuilding = true;
+
+  NIX_CFLAGS_COMPILE = [ "-Wno-format-security" ]
+    ++ lib.optionals stdenv.cc.isClang
+    [ "-Wno-implicit-function-declaration" "-O1" ];
+
+  configureFlags = [
+    "--disable-debug"
+    "--with-bundled-t1lib=yes"
+  ];
+
+  preConfigure = ''
+    configureFlags="$configureFlags --enable-grace-home=$out"
+  '';
```

Comment:
```suggestion
  configureFlags = [ "--enable-grace-home=$out" ];
```

Reply:
Hi,

I used this preconfigure hook as a solution for a specific problem. The variable $out is not evaluated if it's in the configureFlags (of course I use double-quotes). If I add $out in preConfigure it works and is evaluated. Does the usage of preConfigure impose problems?